src/declarations/stencil-private.ts

```diff
@@ -32,16 +34,6 @@ import type {
   VNodeData,
 } from './stencil-public-runtime';
 
-export interface SourceMap {
```

Comment:
since we're depending on the `RawSourceMap` interface from `source-map` now we no longer need our own interface